config/meta/metaImage.js

```diff
@@ -94,13 +94,14 @@ async function generateMetaImage(titleText) {
 
   // add subText text
   ctx.fillStyle = "#fafafa";
-  ctx.font = "italic 48px 'Atkinson Hyperlegible'";
+  // ctx.font = "italic 48px 'Sintony'";
+  ctx.font = "italic 48px 'Sintony'";
```

Comment:
should be normal, not `italic` since Sintony doesn't have italic variant for .ttf